assets/stylesheets/common/question-answer.scss

```diff
@@ -0,0 +1,167 @@
+.qa-answer-count,
+.qa-comments-count {
+  font-size: $font-up-1;
+  box-sizing: border-box;
+}
+
+.qa-comment {
+  font-size: $font-down-1;
+  border-top: 1px solid var(--primary-low);
+
+  .qa-comment-post {
+    padding: 0.7em 0;
+    display: flex;
+    flex-wrap: wrap;
+
+    .qa-comment-cooked {
+      .cooked {
+        padding: 0;
+
+        p {
+          margin: 0;
+        }
+      }
+    }
+
+    .qa-comment-info-separator {
+      margin: 0 0.3em;
+    }
+
+    .qa-comment-info-username {
+      .username {
+        font-weight: 100;
+        margin-right: 0.2em;
+      }
+
+      a {
+        color: var(--tertiary);
+      }
+    }
+
+    .qa-comment-info-created {
+      color: var(--primary-med-or-secondary-med);
+    }
+
+    .qa-comment-actions {
+      display: none;
+      margin-left: auto;
+
+      .qa-comment-actions-edit-link {
+        color: var(--primary-med-or-secondary-med);
+        margin-right: 0.25em;
+      }
+
+      .qa-comment-actions-delete-link {
+        color: var(--primary-med-or-secondary-med);
+      }
+
+      .qa-comment-actions-delete-link:hover {
+        color: var(--danger);
+      }
+    }
+  }
+
+  .qa-comment-editor {
+    margin: 0.5em 0;
+
+    textarea {
+      width: 100%;
+      margin-bottom: 0;
+    }
+
+    .qa-comment-editor-cancel {
+      margin-left: 0.5em;
+    }
+  }
+}
+
+.qa-comment:hover {
+  .qa-comment-actions {
+    display: inline-block;
+  }
+}
+
+.qa-comments-menu {
+  font-size: $font-down-1;
+  margin: 1em 0;
+
+  .qa-comments-menu-composer {
+    width: $topic-body-width;
+
+    .qa-comments-menu-composer-textarea {
+      width: 100%;
+      margin-bottom: 0;
+    }
+
+    .qa-comments-menu-composer-cancel {
+      margin-left: 0.5em;
+    }
+  }
+
+  .qa-comments-menu-seperator {
+    margin: 0 0.8em;
+    display: inline;
+
+    &::after {
+      content: "|";
+    }
+  }
+
+  .qa-comments-menu-show-more,
+  .qa-comments-menu-add-link {
+    display: inline;
+  }
+}
+
+.qa-post {
+  .qa-post-list {
+    box-shadow: shadow("card");
```

Comment:
Btw, is this valid?

Reply:
Yea this is a SASS function in core: https://github.com/discourse/discourse/blob/357186ab7ebd100c38d4776e9e05e9d2d988f570/app/assets/stylesheets/common/foundation/variables.scss#L147-L149